pwa/src/components/badgeCounter/BadgeCounter.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from "react";
+import * as styles from "./BadgeCounter.module.css";
+import clsx from "clsx";
+
+interface BadgeCounterProps {
+  number: number | string;
```

Comment:
I think this component may just force what it wants as a prop, so either a `number` or a `string`, not both.

In this case, I'd opt for a `string`, since we're not doing any computations on it.